components/hab/src/command/studio/docker.rs

```diff
@@ -278,7 +278,7 @@ where
     cmd_args.push("--name".into());
     cmd_args.push(name.into());
     match args.first().map(|f| f.to_str().unwrap_or_default()) {
-        Some("build") => {}
+        Some("build") | Some("run") => {}
```

Comment:
This was already broken, but these cases won't match if the `hab studio` command uses the aliases that are allowed for `build` (`b`, `bu`, `bui`, `buil`) and `run` (`r`, `ru`).

As long as we're changing code, this might be a good time to just take those aliases out.

Reply:
ok. removed.

Reply:
Ooh, I love it when our UX gets better